app/views/hub/coalitions/_form.html.erb

```diff
@@ -0,0 +1,26 @@
+<div>
+  <%= form_with model: @coalition, url: url, method: local_assigns[:http_method], local: true, builder: VitaMinFormBuilder, id: "coalition-form" do |f| %>
+    <h1 class="form-card__title">
+      <%= @title %>
+    </h1>
+
+    <%= f.cfa_input_field :name, t("general.name") %>
+      <div class="form-group state-select">
+        <label for="state-select" class="form-question"><%= t("general.state") %></label>
+        <div>
+          <%= text_field_tag("state_routing_targets[states]",
+                             @coalition.state_routing_targets.map { |srt| States.name_for_key(srt.state_abbreviation) }.to_json.html_safe,
```

Comment:
It seems like we're converting from state abbreviation to full name of the state and vice versa pretty often. Maybe we can add a method to the StateRoutingTarget class.

`@coalition.state_routing_targets.map { |srt| srt.full_state_name }`